### ISSUE TITLE
[Re] Bootstrap Your Own Latent: A new approach to self-supervised learning

### ISSUE THREAD

**Issue by ADevillers** [ReScience/submissions#77]
**Original article:** J.-B. Grill, F. Strub, F. Altché, C. Tallec, P. H. Richemond, E. Buchatskaya, C. Doersch, B. A. Pires, Z. D. Guo,
and M. G. Azar. “Bootstrap your own latent: A new approach to self-supervised learning.” In: arXiv preprint
arXiv:2006.07733 (2020)

**PDF URL:** https://github.com/ADevillers/BYOL/blob/main/report.pdf
**Metadata URL:** https://github.com/ADevillers/BYOL/blob/main/report.metadata.tex
**Code URL:** https://github.com/ADevillers/BYOL/tree/main

**Scientific domain:** Representation Learning
**Programming language:** Python
**Suggested editor:** @rougier

**Comment by rougier** [comment]
Thansk for your submission and sorry for the delay, we'll assign an editor soon.

**Comment by rougier** [comment]
@gdetor @benoit-girard @koustuvsinha Can any of you edit this submission?

**Comment by benoit-girard** [comment]
I can handle this one too (I'll try not to mix up between the two consecutive submissions...).

**Comment by benoit-girard** [comment]
@ADevillers : given the fact that you results were obtained using HPC, that the reviewers may not have access to, are there ways to test you code without it (like running only a subpart of it...)?

(same question applies to the companion paper)

**Comment by ADevillers** [comment]
Yes, you can train on CIFAR10 or perform linear evaluation on CIFAR10 and ImageNet with the following setup:

1. Ensure you use `--computer='other'`.
2. For GPU usage (recommended for faster processing), set `--hardware='mono-gpu'`. If you're using a CPU, set `--hardware='cpu'`.
3. Follow the commands provided in the README's final section.

Note: Training on ImageNet without HPC is impractical due to the large memory requirements for the necessary batch size and the extended duration it might take; yet, evaluation may be fine.

**Comment by benoit-girard** [comment]
Good news: @charlypg has accepted to review this paper and its companion!
